Multisig/UI/Transactions/TransactionsView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TransactionsView.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/27/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionsView: View {
+
+    @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
+    var selectedSafe: FetchedResults<Safe>
+
+    var body: some View {
+        let safe = selectedSafe.first
+        return ZStack {
+            Rectangle()
+               .edgesIgnoringSafeArea(.all)
+               .foregroundColor(Color.gnoWhite)
+            if safe == nil {
```

Comment:
I'd just use `selectedSafe.first` instead of `safe` here and the whole thing will not need the explicit return

Reply:
Then when we need to define the content view we pass selectedSafe.first again ?

Reply:
yes, or define it as a computed property on the `TransactionView` struct